.github/workflows/microk8s-tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         sudo systemctl restart docker
         docker system info
 
-    - uses: balchua/microk8s-actions@e99a1ffcd3bb2682d941104cf6c1a215c657903f
+    - uses: balchua/microk8s-actions@v0.4.3
```

Comment:
Can we please use commit sha for non-Canonical and non-Microsoft/Github actions please.

Reply:
done